packages/pds-ember/package.json

```diff
@@ -2,21 +2,89 @@
   "name": "@hashicorp/pds-ember",
   "version": "0.0.10",
   "description": "Ember addon for building Structure-styled UIs",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/hashicorp/structure.git"
-  },
   "keywords": [
     "ember-addon"
   ],
-  "author": "Ryan Johnson",
   "license": "MPL-2.0",
+  "author": "Ryan Johnson",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/hashicorp/structure.git"
+  },
   "bugs": {
     "url": "https://github.com/hashicorp/structure/issues"
   },
-  "homepage": "https://github.com/hashicorp/structure#readme"
+  "homepage": "https://github.com/hashicorp/structure#readme",
+  "directories": {
+    "doc": "doc",
+    "test": "tests"
+  },
+  "scripts": {
+    "build": "ember build --environment=production",
+    "lint:hbs": "ember-template-lint .",
+    "lint:js": "eslint .",
+    "start": "ember serve",
+    "test": "npm-run-all lint:* test:*",
+    "test:ember": "ember test",
+    "test:ember-compatibility": "ember try:each",
+    "lint-all": "yarn lint:hbs && yarn lint:js",
+    "g:component": "ember g pds-component",
+    "d:component": "ember d pds-component",
+    "storybook": "start-storybook -p 6006 -s dist",
+    "storybook:build": "build-storybook"
```

Comment:
These scripts were pulled from HCP addon, but may not function until an upcoming PR imports source files from HCP.